include/constants/daycare.h

```diff
@@ -0,0 +1,32 @@
+#ifndef GUARD_DAYCARE_CONSTANTS_H
+#define GUARD_DAYCARE_CONSTANTS_H
+
+// Egg parents
+#define EGG_MOTHER 0
```

Comment:
`EGG_MOTHER` and `EGG_FATHER` seem to be misnomers.  They are really just slots and have nothing to do with which parent it is.

Reply:
The mother is inherently slot 0, see something like `DetermineEggSpeciesAndParentSlots` where only the the first daycare mon is used to determine species. It is sort of misleading though, because the indices are referred to regardless (like if there isn't an egg, or there's only one mon in daycare). Should they just be dropped altogether?

Reply:
The constants you've defined are only relevant for a single line of code, and they don't make sense outside of it, so they probably should be removed altogether.
```
BuildEggMoveset(&egg, &daycare->mons[parentSlots[1]].mon, &daycare->mons[parentSlots[0]].mon);
```

Reply:
I guess they're also relevant in this code:
```
u8 ditto = parentSlots[EGG_FATHER];
parentSlots[EGG_FATHER] = parentSlots[EGG_MOTHER];
parentSlots[EGG_MOTHER] = ditto;
```

However, the constants are pretty misleading in general, especially in the scripts.

Reply:
Yeah outside of `DetermineEggSpeciesAndParentSlots` they don't make a ton of sense. They sort of do in `GetDaycareCompatibilityScore` but the parents are interchangeable there so it doesn't matter.